src/murmur/Server.h

```diff
@@ -409,6 +410,7 @@ class Server : public QThread {
 		void readChannelPrivs(Channel *c);
 		void setLastChannel(const User *u);
 		int readLastChannel(int id);
+		void setLastSeen(const User *u);
```

Comment:
Please add a documentation String (using Doxygen style `///` comments) to this function